model/training.py

```diff
@@ -20,16 +21,28 @@
 labels_filename = "labels.txt"
 
 
-def parse_args():
+def parse_args(args):
     """Returns dataset file, model output directory, and num_epochs if present. These must be parsed as command line
     arguments and then used as the model input and output, respectively. The number of epochs can be used to optionally override the default.
     """
     parser = argparse.ArgumentParser()
     parser.add_argument("--dataset_file", dest="data_json", type=str)
     parser.add_argument("--model_output_directory", dest="model_dir", type=str)
     parser.add_argument("--num_epochs", dest="num_epochs", type=int)
-    args = parser.parse_args()
-    return args.data_json, args.model_dir, args.num_epochs
+    parser.add_argument(
+        "--labels",
+        dest="labels",
+        type=str,
+        required=False,
+        help="Space-separated list of labels",
```

Comment:
Maybe worth saying here that they must be encapsulated by single quotes and leave a comment with an example? Also worth updating the README with instructions on how to submit the job

Reply:
bumping this! for both this & the classification example